.github/workflows/test.yml

```diff
@@ -132,16 +132,29 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1.18.x
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        role-to-assume: ${{ secrets.TERRAFORM_PROVIDER_ROLE }}
+        aws-region: us-east-1
     - name: Import GPG key
       id: import_gpg
-      uses: crazy-max/ghaction-import-gpg@v2
-      env:
-        GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      uses: crazy-max/ghaction-import-gpg@v5
+      run: |
```

Comment:
Does this work? My understanding was that `uses` was mutually exclusive with `run`. Although I can't find any docs that support that. You may need to split this up into a first run step and then an import step.

What did you do to test this?

Reply:
That is right. You cannot use `run` and `use` in same step

Reply:
Thanks @gaiksaya and @phillbaker, just modified the PR.